src/main/java/eiteam/esteemedinnovation/armor/ArmorModule.java

```diff
@@ -88,7 +88,7 @@
 import static net.minecraft.init.Items.*;
 
 public class ArmorModule extends ContentModule implements ConfigurableModule {
-    public static final ItemArmor.ArmorMaterial STEAM_EXO_MAT = EnumHelper.addArmorMaterial("STEAMEXOSUIT", MOD_ID + ":steam_exo", 15, new int[] { 2, 5, 4, 1 }, 0, SoundEvents.ITEM_ARMOR_EQUIP_GENERIC, 0F);
```

Comment:
There should be a space between `[]` and `{`. The ones inside are optional as defined in the code style, but perhaps that should be changed to be one or the other.